client/src/pages/Profile/Profile.tsx

```diff
@@ -1,7 +1,132 @@
-export default function Profile(): JSX.Element {
+import {
+  AppBar,
+  Avatar,
+  Button,
+  Box,
+  Grid,
+  Paper,
+  Tab,
+  Tabs,
+  Typography,
+  ThemeProvider,
+  createMuiTheme,
+} from '@material-ui/core';
+import { useEffect, useState } from 'react';
+import { User } from '../../interface/User';
+import { Link } from 'react-router-dom';
+import useStyles from './useStyles';
+import { contests } from '../../interface/tempContestData';
+import ListView from '../../components/ListView/ListView';
+import { contest } from '../../interface/tempContestData';
+
+interface Props {
+  loggedIn: boolean;
+  user: User;
+}
+
+export default function Profile({ user }: Props): JSX.Element {
+  const classes = useStyles();
+  const [value, setValue] = useState(0);
+  const conts = contests;
+  const [newContest, setNewContest] = useState<contest[]>(Object);
+
+  const MyTheme = createMuiTheme({
+    palette: {
+      primary: {
+        main: '#000000',
+      },
+      secondary: {
+        main: '#3d3d3d',
+      },
+    },
+  });
+
+  interface TabPanelProps {
+    children?: React.ReactNode;
+    dir?: string;
+    index: number;
+    value: number;
+  }
+
+  function TabPanel(props: TabPanelProps) {
+    const { children, value, index, ...other } = props;
+
+    return (
+      <div
+        role="tabpanel"
+        hidden={value !== index}
+        id={`full-width-tabpanel-${index}`}
+        aria-labelledby={`full-width-tab-${index}`}
+        {...other}
+      >
+        {value === index && (
+          <Box p={3}>
+            <Typography>{children}</Typography>
+          </Box>
+        )}
+      </div>
+    );
+  }
+
+  //Reduces the collection of contest to contests that involves the user
+  const handleContest = () => {
+    setNewContest(conts.filter((e) => e.creator.indexOf(user.email) !== -1));
```

Comment:
maybe make a note of here that filtering won't be the best choice here to do on the frontend. when we have real data, it should be done on the backend!

Reply:
You are absolutely right I will append that